src/styles/_footer.scss

```diff
@@ -1,19 +1,234 @@
-/***** Footer *****/
-.footer {
-  border-top: 1px solid $border-color;
-  margin-top: 60px;
-  padding: 30px 0;
+@use './mixins';
 
-  a { color: $secondary-text-color; }
+/***** Footer *****/
+@media only screen and (min-width: 320px) {
+  .Footer-module--container {
+    grid-template-areas: 'a b' 'c d' 'branding branding' 'copyright copyright';
+  }
 
-  &-inner {
-    @include max-width-container;
+  .Footer-module--branding {
     display: flex;
     justify-content: space-between;
   }
 
-  &-language-selector button {
-    color: $secondary-text-color;
+  .Footer-module--logo {
```

Comment:
`.footer` instead of `.Footer-module` would be more appropriate.  dev.maxmind.com uses `Footer-module` because it's using a postcss plugin that autogenerates css class names.

Reply:
@kevcenteno thanks, I'll update those accordingly. Is there a preference between having the double dash in the class names as well? In other words, should we keep it `footer--logo` or should it be `footer-logo`?

Reply:
The double dashes are nice, double underscores even nicer.  [BEM](http://getbem.com/naming/) is a decently popular selector naming convention